### ISSUE TITLE
Feature Request Default Sort

### ISSUE THREAD

**Issue by randallmeeker** [balderdashy/waterline#486]
should be able to set a default sorting option when defining your model. I don't make feature requests lightly, because I know everyone has their own. However, I can't imagine ever running a find action that does not require a sort and that sort being the same almost every time. Am I wrong?

**Comment by mikermcneil** [comment]
Thanks for posting, @randallmeeker.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/waterline/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
